### PR TITLE
Gradient

### PR DESCRIPTION
**Pull request by Charu271**
Issue:

### CODE REVIEW

File: src/_variables.less [hunk 1]
```diff
@@ -76,6 +76,16 @@
 @keyBoxWhite: #bdb6b6;
 @diffblack: #686363;
 @diffwhite: #fdf4f4;
+@gradientPink: #e60550;
```
Comment (shahednasser) [inline src/_variables.less:79]: See the "Begin variables IN styles/_key.less" and "END VARIABLES IN styles/_key.less"? Move your variables from in between those comments and add before your variables similar comments before and after them but change _key.less to _gradient.less

**Review by shahednasser — CHANGES_REQUESTED** [review]
Please do the changes in the review and run `npm run compile` then commit the changes.

**Comment by Charu271** [comment]
Mam, I am not able to push changes to this pull request. Shall I make a new pull request?

**Comment by shahednasser** [comment]
Thank you!

@all-contributors please add @Charu271 for code

**Comment by Charu271** [comment]
Thanks mam. Can u please add a label of hacktober fest accepted

**Comment by shahednasser** [comment]
Hi @Charu271 don't worry, this repository has the "hacktoberfest" topic so all merged pull requests are accepted.

**Comment by Charu271** [comment]
Okay
Thank you mam

**Pull request merged**